.github/workflows/nightly-release.yml

```diff
@@ -1,31 +1,35 @@
-name: Nightly Releases
+name: nightly releases
 
-#on:
-#  schedule:
-#    # Run at 00:00 UTC Every Day
-#    - cron: "0 0 * * *"
-
-on: workflow_dispatch
+on:
+  schedule:
+    # Run at 00:00 UTC Every Day
+    - cron: "0 0 * * *"
 
 jobs:
   version-override:
     runs-on: ubuntu-latest
-    name: Build and Publish PyPi binaries
     steps:
       - uses: actions/checkout@v2
+      - name: Set Git config
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "Github Actions"
       - name: Rebase with main and update version
         run: |
+          git fetch --unshallow
           git checkout main
           git pull --rebase
           git checkout nightly
-          git merge main
+          git pull --rebase
+          git merge main --strategy-option theirs
+          git checkout main -- ./aim/VERSION
           . ./.github/workflows/override-version.sh
```

Comment:
while looking into this yesterday evening found some repos use `./.github/scripts/` as a path for such utilities. I think it's a good pattern.

Reply:
will move :)